CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# [0.11.0](https://github.com/ComplianceAsCode/auditree-arboretum/releases/tag/v0.11.0)
+
+- [ADDED] Organization repository permissions fetcher added to `permissions`.
```

Comment:
Can you add more detail to your `[ADDED]` entry and add a `[DEPRECATED]` entry for the org repo collabs fetcher and state that its functionality is now part of the permissions fetcher?  Something like:

```
[ADDED] Github org integrity fetcher added to `permissions`.
[ADDED] Github org integrity forks fetcher functionality added Github org integrity fetcher.
[DEPRECATED] Github org integrity collaborators fetcher functionality moved to Github org integrity fetcher. 
```